scylla-cdc/src/log_reader.rs

```diff
@@ -214,3 +162,147 @@ impl CDCReaderWorker {
             .await;
     }
 }
+
+pub struct CDCLogReaderBuilder {
+    session: Option<Arc<Session>>,
+    keyspace: Option<String>,
+    table_name: Option<String>,
+    start_timestamp: chrono::Duration,
+    end_timestamp: chrono::Duration,
+    window_size: time::Duration,
+    safety_interval: time::Duration,
+    sleep_interval: time::Duration,
+    readers: Vec<Arc<StreamReader>>,
```

Comment:
The readers field is unnecessary. The new CDCLogReaderBuider should start with no readers.